nix/nim-status/default.nix

```diff
@@ -0,0 +1,150 @@
+{ newScope
+, mkShell
+, writeScript
+, writeText
+, xcodeWrapper
+, pkgs
+, stdenv
+, lib }:
+let
+  callPackage = newScope {};
+  buildNimStatus = platform: arch: callPackage ./nim-status.nix {
+    inherit platform arch;
+  };
+
+  # Metadata common to all builds of status-go
+  meta = {
+    description = "The Status Go module that consumes go-ethereum.";
+    license = lib.licenses.mpl20;
+    platforms = with lib.platforms; linux ++ darwin;
+  };
```

Comment:
I'm not seeing this `meta` block being used anywhere. I see it's being passed to `nix/nim-status/status-go.nix` but not used. Might as well just move it there directly.

Reply:
Moved it to status-go and fixed the build code so that it's actually used.